### PR TITLE
First draft of "10 Myths about Software Testing"

### PR DESCRIPTION
**Pull request by hxmattobee**
### Blog title
10 Myths about Software Testing

### Blog description
Following on loosely from my last post, this busts 10 of the most common myths in software testing.

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by LouisPryer** [comment]
This is a good read @hxmattobee, I would suggest possibly adding in an "In Summary" section near the end but I love the style. Almost like a 10 commandments for testing! Also, possibly creating a link to ISTQB website would be a nice touch.

**Comment by hxmattobee** [comment]
I've addresses those two points, thanks @LouisPryer.

File: _posts/2015-10-05-10-myths-about-software-testing.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+This is true for <em>some</em> projects, but certainly not all. In highly-regulated industries or where the level of risk is high (e.g. healthcare, finance) testing does need to be more structured and formalised. For other projects, particularly those following an Agile workflow, there simply isn't time for lots of advanced planning. Exploratory testing is all about simultaneous test design and execution, drawing on the skill and experience of the tester instead of a pre-written plan. That said, while we can't plan each individual test, we should plan the scope and general approach of our testing sessions.
+
+## 3. Exploratory testing is bad testing
+
+This myth comes from the assumption that <em>exploratory</em> means the same thing as <em>random</em>. In fact, exploration is a logical and systematic approach to testing. All testing that involves a person interacting, thinking and responding to what happens is inherently exploratory, but the amount of freedom they have might be limited by things like overly prescriptive test cases. Exploratory testing often reveals problems and opportunities that more formalised testing and checking cannot.
+
```
Comment (alannamooney) [inline _posts/2015-10-05-10-myths-about-software-testing.md]: Might read better as 'All testing involves a person interacting with, thinking about, and responding to what happens is inherently exploratory'?

**Comment by alannamooney** [comment]
Have taken editor 1 and added a quick note around a potential wording change. Other than that it all looks great - very interesting read :-)

**Comment by alannamooney** [comment]
+1 from me :)

**Comment by jackdcrawford** [comment]
Taking editor 2 :raised_hands:

**Comment by jackdcrawford** [comment]
Great read @hxmattobee, really enjoyed it. :shipit:

**Pull request merged**